clients/src/main/java/org/apache/kafka/clients/consumer/MockConsumer.java

```diff
@@ -92,11 +92,37 @@ public synchronized Set<TopicPartition> assignment() {
         return this.subscriptions.assignedPartitions();
     }
 
-    /** Simulate a rebalance event. */
+    /**
+     * Simulate a rebalance event.
+     */
     public synchronized void rebalance(Collection<TopicPartition> newAssignment) {
-        // TODO: Rebalance callbacks
+        // prepare for rebalance callback
```

Comment:
This is more like, computing the partitions to be added and removed, right?

Reply:
thanks - updated